sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/PeriodicSequence.java

```diff
@@ -178,9 +182,22 @@ public RestrictionTracker<OffsetRange, Long> newTracker(@Restriction OffsetRange
       return new OutputRangeTracker(restriction);
     }
 
+    @GetInitialWatermarkEstimatorState
+    public Instant getInitialWatermarkState() {
+      return BoundedWindow.TIMESTAMP_MIN_VALUE;
```

Comment:
Should we use the restriction's minimum value here? (or maybe slightly less than that to allow the first element to be emitted)